### DISCUSSION TITLE
Human Readable Date in the Added column

### DISCUSSION THREAD

**Discussion by artik** [jesec/flood#350] [Ideas]
Something that bothers me is the fact that I can't properly count the day a `.torrent` file as been added. I would be more natural to have a `human readable date`  in the `Added` column.

Instead of `7/1/2021` having `2 days ago` till 31 days ago. Then, use the classic date format.

**Comment by C84186** [comment]
That's a nice idea. There are probably plenty of packages that do exactly that. 
I'd suggest making it a separate column, though, rather than replacing date. 

Personally, my gripe with `Added` is exactly the opposite, it only gives a date & no time.

**Reply by artik** [reply]
I don't think creating a new column, for the same, info makes sense. Maybe a setting in the options you can switch to display the format you prefer ?

**Reply by C84186** [reply]
I disagree. Toggling and moving columns in Flood is intuitive, discoverable, and low complexity, while adding a toggle for time-display preference adds an extra element, also - there are instances where it is useful to be able to see both the humane time-delta, as well as the exact time you added a torrent.

For the purposes of reducing complexity in the settings, I'd argue that it's probably easier to add a column for "Duration since added" that users can toggle, and if they prefer, they can also toggle off "Date added"

**Reply by artik** [reply]
up to you guys. One way or the other is fine as soon as I can read properly the data of my torrents. I'd do it with a toggle in the settings, as you can change `date format`, etc.
